src/inmanta/protocol/methods.py

```diff
@@ -1005,6 +1005,17 @@ def get_server_status() -> model.StatusResponse:
     """
 
 
+@method(
```

Comment:
typed would be beter, if not fixed now, make a ticket so we don't forget.

Reply:
#2279